### PR TITLE
Adcm-2056 add ui tests for back button in browser

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/test_header_and_footer.py [hunk 1]
```diff
@@ -31,80 +32,121 @@
...
+            hosts_page = HostListPage(app_fs.driver, app_fs.adcm.url)
+            hosts_page.wait_page_is_opened()
+            hosts_page.driver.back()
+            hostprovider_page.wait_page_is_opened()
+        with allure.step("Check back button for jobs page header link"):
+            hosts_page.open()
+            hosts_page.header.click_jobs_tab_in_header()
+            jobs_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+            jobs_page.wait_page_is_opened()
+            jobs_page.driver.back()
```
Comment (skhomuti) [inline tests/ui_tests/test_header_and_footer.py:133]: Add allure step to this method

Comment (skhomuti) [reply tests/ui_tests/test_header_and_footer.py:133]: Probably you can add it as a BasePageObject method

Comment (annademina) [reply tests/ui_tests/test_header_and_footer.py]: added

**Review by skhomuti — APPROVED** [review]

File: tests/ui_tests/test_header_and_footer.py [hunk 2]
```diff
@@ -31,80 +32,121 @@
...
+            header.click_doc_link_in_help_popup()
+        with allure.step(f"Check new opened page: {params['docs_link']}"):
+            BasePageObject(app_fs.driver, app_fs.adcm.url).wait_url_contains_path(params["docs_link"])
+
+    def test_check_header_user_settings_for_authorised_user(self, app_fs):
+        header = PageHeader(app_fs.driver, app_fs.adcm.url)
+        header.click_account_button_in_header()
+        header.check_account_popup()
+        header.click_settings_link_in_acc_popup()
+        header.wait_url_contains_path(AdminSettingsPage(app_fs.driver, app_fs.adcm.url).path)
```
Comment (Sealwing) [inline tests/ui_tests/test_header_and_footer.py:94]: Why like that and not wait until the page is opened?

**Review by Sealwing — APPROVED** [review]

File: tests/ui_tests/test_header_and_footer.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/test_header_and_footer.py:94]: it was the first implementation and now I just moved these tests into class and didn't change inside

Comment (Sealwing) [reply tests/ui_tests/test_header_and_footer.py:94]: Got it

**Pull request merged**